a10_octavia/controller/worker/flows/a10_member_flows.py

```diff
@@ -968,6 +968,295 @@ def get_batch_update_member_snat_pool_subflow(self, member):
+                    name='member-delete' + m.id,
+                    inject={constants.MEMBER: m},
+                    requires=(
+                        constants.MEMBER,
+                        a10constants.VTHUNDER,
+                        constants.POOL,
+                        a10constants.MEMBER_COUNT_IP,
+                        a10constants.MEMBER_COUNT_IP_PORT_PROTOCOL)))
+            batch_update_members_flow.add(database_tasks.DeleteMemberInDB(
+                name='delete-member-in-db' + m.id,
+                inject={constants.MEMBER: m}))
+            batch_update_members_flow.add(database_tasks.DecrementMemberQuota(
+                name='decrement-member-quota' + m.id,
+                inject={constants.MEMBER: m}))
+        batch_update_members_flow.add(
+            self.get_delete_member_vrid_internal_subflow(constants.POOL, old_members))
+
+        # for creation of members
+        batch_update_members_flow.add(lifecycle_tasks.MembersToErrorOnRevertTask(
+            name='{flow}-created'.format(
+                flow=constants.MEMBER_TO_ERROR_ON_REVERT_FLOW),
+            inject={constants.MEMBERS: new_members},
+            requires=[constants.LISTENERS,
+                      constants.LOADBALANCER,
+                      constants.POOL]))
+        for m in new_members:
+            batch_update_members_flow.add(database_tasks.MarkMemberPendingCreateInDB(
+                name='mark-member-pending-create-in-DB' + m.id,
+                inject={constants.MEMBER: m}))
+            batch_update_members_flow.add(a10_database_tasks.CountMembersWithIP(
+                name='count-members-with-ip' + m.id,
+                inject={constants.MEMBER: m},
+                provides=a10constants.MEMBER_COUNT_IP))
+            batch_update_members_flow.add(vthunder_tasks.AllowLoadbalancerForwardWithAnySource(
+                name=a10constants.ALLOW_NO_SNAT + m.id,
+                inject={constants.MEMBER: m},
+                requires=(constants.AMPHORA)))
+
+            batch_update_members_flow.add(server_tasks.MemberCreate(
+                name='member-create' + m.id,
+                inject={constants.MEMBER: m},
+                requires=(constants.MEMBER, a10constants.VTHUNDER, constants.POOL,
+                          a10constants.MEMBER_COUNT_IP, constants.FLAVOR)))
+            batch_update_members_flow.add(database_tasks.MarkMemberActiveInDB(
+                name='mark-active-in-DB' + m.id,
+                inject={constants.MEMBER: m}))
+
+        # for updating of members
+        batch_update_members_flow.add(
+            lifecycle_tasks.MembersToErrorOnRevertTask(
+                name='{flow}-updated'.format(
+                    flow=constants.MEMBER_TO_ERROR_ON_REVERT_FLOW),
+                # updated_members is a list of (obj, dict), only pass `obj`
+                inject={constants.MEMBERS: [m[0] for m in updated_members]},
+                requires=[constants.LISTENERS,
```

Comment:
Please remove the listener, pool, and lb from this task, [similar to Octavia code]

Reply:
DONE